lib/jackal-slack/notification.rb

```diff
@@ -14,7 +14,9 @@ def setup(*_)
       # @return [Truthy, Falsey]
       def valid?(message)
         super do |payload|
-          payload.get(:data, :slack)
```

Comment:
@chrisroberts should this change be sufficient for making this callback behave as we discussed (for no-oping when missing relevant configs)?